openapi.yaml

```diff
@@ -0,0 +1,669 @@
+          description: The refresh interval on which a sync should be initiated (while viewing the app). In milliseconds.
+          type: number
+          example: 60000
+    
+    create-pinned-search:
+      title: Post or Patch Pinned Search
+      type: object
+      properties:
+        id:
+          type: number
+          description: The search ID.
+          example: 35778
+        user_id:
+          type: number
+          description: The User ID.
+          example: 11741
+        query:
+          type: string
+          description: The search query.
+          example: "owner:octobox inbox:true"
+        name:
+          type: string
+          description: The search name.
+          example: "Work"
+        count:
+          type: number
+          description: The count.
+          example: 0
+        created_at:
+          type: string
+          description: The date and time of creation.
+          example: "2021-11-23T19:48:39.953Z"
+        updated_at:
+          type: string
+          description: The date and time of update.
+          example: "2021-11-23T19:48:39.953Z"
+          
+    return-pinned-searches:
+      title: Return Pinned Searches
+      type: object
+      properties:
+        pinned_searches:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: integer
+              user_id:
+                type: integer
+              query:
+                type: string
+              name:
+                type: string
+              count:
+                type: integer
+              created_at:
+                type: string
+              updated_at:
+                type: string
+                
+    return-pinned-search:
+      title: Return a Pinned Search
+      type: object
+      properties:
+        id:
+          type: integer
+        user_id:
+          type: integer
+        query:
+          type: string
+        name:
+          type: string
+        count:
+          type: integer
+        created_at:
+          type: string
+        updated_at:
+          type: string
+          
+          
+    notifications:
+      title: Notifications
+      type: object
+      properties:
+        pagination:
+          type: object
+          properties:
+            total_notifications:
+              type: integer
+            page:
+              type: integer
+            total_pages:
+              type: integer
+            per_page:
+              type: integer
+        types:
+          type: object
+          properties:
+            PullRequest:
+              type: integer
+        reasons:
+          type: object
+          properties:
+            mention:
+              type: integer
+        unread_repositories:
+          type: object
+          properties:
+            octobox/octobox:
+              type: integer
+        notifications:
+          type: array
+          items:
+            type: object
```

Comment:
Can we pull this object out into it's own `Notification` Object and do:

```
items:
  "$ref": "#/components/schemas/Notification"
```

here instead